src/App.tsx

```diff
@@ -46,6 +46,9 @@ const App = () => {
       <AuthenticatedRoute path="/scan" exact>
         <ScanPage />
       </AuthenticatedRoute>
+      <AuthenticatedRoute path="/admin/create-users" exact requiredPermission="BULK_CREATE_USERS">
```

Comment:
What do you think of the path and required permission name?

cc @StevePole @Tankenstein

Reply:
Love it